### PR TITLE
[Docs] Add examples in swagger docs

### LINKED ISSUE #4968: [Docs] Examples of of OAM endpoints
@utkarsh-pro thanks for clarifying. Examples of endpoints are ideal; a set of examples that provide something of a template such that a contributor could plugin the name of an object (by taking a guess) and hit the endpoint even though not all endpoints are exhaustively listed in the docs.

_Originally posted by @leecalcote in https://github.com/meshery/meshery/pull/4943#discussion_r786280208_

### PR DESCRIPTION
**Pull request by tangledbytes**
Signed-off-by: Utkarsh Srivastava <srivastavautkarsh8097@gmail.com>

**Description**

This PR fixes #4968 

**Notes for Reviewers**

**[Signed commits](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by utkarsh-pro** [comment]
@leecalcote is this what we were looking for?

File: handlers/oam_handler.go [hunk 1]
```diff
@@ -309,6 +321,7 @@ func (h *Handler) POSTOAMRegisterHandler(typ string, r *http.Request) error {
 // Getting list of workloads/traits/scopes
 //
 // {type} being of either trait, scope, workload; registration of adapter capabilities.
+// Example: /api/oam/workload => Here {type} is "workload"
```
Comment (leecalcote) [inline handlers/oam_handler.go:324]: Thank you, @utkarsh-pro. This is a good example. Will you also show how a specific set of workloads can be retrieved, using an adapter as an example? How to filter to only a single adapter? How to filter to only certain types (maybe those having a specific attribute) of workflows for a given adapter?

**Review by leecalcote — CHANGES_REQUESTED** [review]

File: handlers/oam_handler.go (hunk 1 above)
Comment (leecalcote) [reply handlers/oam_handler.go:324]: Something like this: `http://localhost:9081/api/oam/workload/adapter.meshery.io/meshery-istio` -- how to pull only the definitions for a specific registered component?

Comment (utkarsh-pro) [reply handlers/oam_handler.go:324]: @leecalcote this is covered in the doc of another endpoint.

Comment (utkarsh-pro) [reply handlers/oam_handler.go:324]: @leecalcote also, the endpoint (at the moment) do not support advanced querying as the initial case was quite simple, that was to minimise outputs of other endpoint.

Comment (leecalcote) [reply handlers/oam_handler.go:324]: I've missed it, which other endpoint are you referring to?

So, is this as deep as the endpoint goes? `http://localhost:9081/api/oam/workload/` or can something more specific be queried? 

// @Revolyssup

**Pull request merged**